integrations/artifactory_buildinfo.rst

```diff
@@ -0,0 +1,63 @@
+.. _integrations_artifactory_build_info:
+
+Artifactory Build Info
+======================
+
+The `Artifactory build info <https://www.buildinfo.org/>`_ Artifactory build info is a recollection of the metadata of a build.
+This json-formatted file includes all the details about the build broken down into segments like version history, artifacts, project
+modules, dependencies, and everything that was required to create the build.
+
+Build infos are identified with a ``build name`` and a ``build number``, similar to how many CI services identify the builds.
+They are conveninently stored in Artifactory to keep track of the build metadata and to perform different operations like merging
+the metadata with other build infos or build promotion between repositories.
```

Comment:
This sentence made me think that you can merge the BuildInfo's inside Artifactory, but you can't, can you? Also, as promotions don't work directly with BuildInfo's for Conan and we will do those using Release Bundles I think it's better not to mention them. What do you think?

Reply:
ok, let's skip this part for the moment